src/ast/mod.rs

```diff
@@ -9203,6 +9232,30 @@ pub enum CopyIntoSnowflakeKind {
     Location,
 }
 
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct ReturnStatement {
+    pub value: Option<ReturnStatementValue>,
+}
+
+impl fmt::Display for ReturnStatement {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match &self.value {
+            Some(ReturnStatementValue::Expr(expr)) => write!(f, "RETURN {}", expr),
+            None => write!(f, "RETURN")
+        }
+    }
+}
+
+/// Variants of the Mssql `RETURN` statement
```

Comment:
```suggestion
/// Variants of a `RETURN` statement
```
since this statement isn't mysql specific